airflow/contrib/example_dags/example_gcp_dataproc_create_cluster.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+Example Airflow DAG that creates DataProc cluster.
+"""
+
+import airflow
+from airflow import models
+from airflow.contrib.operators.dataproc_operator import DataprocClusterCreateOperator
+
+default_args = {"start_date": airflow.utils.dates.days_ago(1)}
+
+cluster_name = "testcluster-800"
+project_id = ""
+region = ""
+zone = ""
+
+# When using autoscaling the number of primary workers
+# must be within autoscaler min/max range
+num_workers = 2
+
+# Autoscaling policy
+# The policy must be in the same project and Cloud Dataproc region
+# but the zone doesn't have to be the same
+scaling_policy = 'test-policy'
+policy_uri = 'projects/{p}/locations/{r}/autoscalingPolicies/{id}'.format(
```

Comment:
```suggestion
POLICY_NAME = 'projects/{project_id}/locations/{region}/autoscalingPolicies/{autoscaling_policy_id}'.format(
```
I think it's worth creating a static method in the hook from this code snippet.

Reply:
For sure the short variable names are too short :). I prefer readable names (and this is what pylint expects)

Reply:
I think that I will remove the autoscaling policy from the example for now. It should be added with proper operator (`DataprocCreatePolicy` or something) making the example fully functional:
`create_policy >> create_cluster >> delete_cluster`